pylintrc

```diff
@@ -115,6 +115,7 @@ disable =
     wrong-import-position,
     no-name-in-module,
     locally-disabled,
+    locally-enabled,
```

Comment:
pylint warns when you using `#pylint: enable` just as when you do `#pylint: disable`, so this is here to quiet that warning.